docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3.8'
```

Comment:
Let's add DB here (if applicable) and define a separated network for each service.

Ex:
```yml
services:
    postgres:
        image: bitnami/postgresql:latest
        container_name: identity-postgres-container
        environment:
            - POSTGRESQL_DATABASE=Identity
            - POSTGRESQL_USER=identityuser
            - POSTGRESQL_PASSWORD=identityuser
        ports:
            - "5440:5432"
        networks:
            - academic_hub_backend_vpn

    identity:
        build:
            context: .
            dockerfile: Dockerfile.Identity
        restart: no
        container_name: academic_hub_identity
        environment:
            - NAME=VALUE
        ports:
            - $local:$service
        networks:
            - academic_hub_backend_vpn

networks:
    academic_hub_backend_vpn:
        name: academic_hub_backend_vpn
```

Reply:
then show me the error